Tests/Fennec_Enterprise_XCUITests.xctestplan

```diff
@@ -65,7 +65,6 @@
         "HomePageSettingsUITests\/testCustomizeHomepage()",
         "HomePageSettingsUITests\/testJumpBackIn()",
         "HomePageUITest",
-        "IntegrationTests",
```

Comment:
Don't know why this is removed from the skipped tests but it should be as it was. Please don't change this...

Reply:
Oops. I was playing with XCode and learned how to enable and to disable tests. I'm putting this line back! 📝